Formula/o/omniorb.rb

```diff
@@ -36,15 +36,19 @@ def install
     inreplace "configure",
               /am_cv_python_version=`.*`/,
               "am_cv_python_version='#{xy}'"
-    system "./configure", "--prefix=#{prefix}"
-    system "make"
+    if OS.mac?
+      system "./configure", *std_configure_args, "--with-openssl", "--enable-cfnetwork"
+    else
+      system "./configure", *std_configure_args, "--with-openssl"
+    end
+    system "make", "--jobs=#{ENV.make_jobs}"
```

Comment:
`brew` already sets `MAKEFLAGS=-j#{ENV.make_jobs}`, so I don't think this is necessary.
```suggestion
    system "make"
```